### PR TITLE
daemon,client,cmd/snap: move query metadata to top-level doc

### PR DESCRIPTION
**Pull request by niemeyer**

### CODE REVIEW

File: docs/rest.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ Sample result:
     }
  },
  "paging": {
-    "count": 3,
+    "total": 3,
```
Comment (chipaca) [inline docs/rest.md]: "total" might be unknowable, but we can address this when we get there

File: docs/rest.md [hunk 2]
```diff
@@ -267,7 +267,6 @@ Sample result:
       updated to the revision specified as a value to this entry.
     * `channel`: which channel the package is currently tracking.
 * `paging`
-    * `count`: the number of snaps on this page
```
Comment (chipaca) [inline docs/rest.md]: document "total" if you're adding it

**Comment by chipaca** [comment]
retest this please
LGTM

File: docs/rest.md (hunk 1 above)
Comment (niemeyer) [inline docs/rest.md]: Sorry, that was a quick thought leaking into the code.

**Pull request merged**